src/main/resources/io/github/dfdlschemas/envelopepayload/xsd/payloadContentGroup.dfdl.xsd

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is the content of the original envelopePayload.dfdl.xsd with just the root element removed.